content/docs/support-and-ops/processes/ai-workflow-support.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "AI workflow support"
+owner:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+confidentiality: public
+---
+
+## Why do we need this?
+
+We have seen some of the questions from our customers are replied multiple times. At the same time there are different areas in the company and none now knows everything across teams. Ideally to solve that we put everything in docs so we can profit in support as main source of information. But the problem comes when we need to get the exact question replied for the huge amounts of content our docs have. The goal with the `AI support workflow` is to solve this pain, allowing our support colleagues ask question to a GPT (called `Inkeep`) and get a decent correct answer.
+
+## How this works?
+
+We have a workflow that could look like
+
+![AI workflow schema](ai-workflow-00.png)
+
+Here we see there is a bot, called `@Ask Inkeep`, which is [automatically called every new support thread by a dummy reply bot](https://github.com/giantswarm/slack-replier). Inkeep bot will use Open AI technology to query a custom GPT for an answer that has been trained with our official documentation.
+
+THe AI bot will send a request to trained model and will reply in the same thread. Also the bots ask for feedback on the response. This is important to make the responses more accurate. In case the response is good, it can be used to reply the customer directly (potentially one day we can put this in front on the customer). In case it is not, then as usual we ask the owner team to give us a correct answer. It could be there is a back and forth till we mark the thread as done.
```

Comment:
```suggestion
The AI bot will query the trained model and reply in the same thread. The bot also asks for feedback on the response which is important to increase accuracy, over time. In case the response is good, it can be used in a reply to the customer directly (potentially one day we can put this in front on the customer). In case it is not, then we ask the owner team to give us a correct answer, as usual. There may still occur some discussion till we mark the support thread as done.
```